js/runner.go

```diff
@@ -170,6 +170,14 @@ func (r *Runner) newVU(id int64, samplesOut chan<- stats.SampleContainer) (*VU,
 		BlockedHostnames: r.Bundle.Options.BlockedHostnames.Trie,
 		Hosts:            r.Bundle.Options.Hosts,
 	}
+	if r.Bundle.Options.LocalIPs.Valid {
+		ipIndex := uint64(id - 1)
+		if id == 0 {
+			ipIndex = 1
+		}
```

Comment:
This seems incorrect :confused: If I'm reading it right, then we'd get the second IP for `setup()` and `teardown()`, not the first? And it's also a bit confusingly written, this seems simpler:
```suggestion
		var ipIndex uint64
		if id > 0 {
			ipIndex = uint64(id - 1)
		}
```